pkg/config/config.go

```diff
@@ -363,8 +363,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("server_timeout", 30)
 
 	// Configuration for TLS for outgoing connections
-	config.BindEnvAndSetDefault("force_tls_12", false) // deprecated
-	config.BindEnvAndSetDefault("min_tls_version", "") // default depends on force_tls_12
+	config.BindEnvAndSetDefault("force_tls_12", false)        // deprecated
```

Comment:
```suggestion
	config.BindEnvAndSetDefault("force_tls_12", false) // deprecated
```

Reply:
I think these space are coming from the linter to match the comment below 🤷